www-client/links/links-2.20.2.recipe

```diff
@@ -75,10 +72,11 @@ defineDebugInfoPackage links$secondaryArchSuffix \
 
 BUILD()
 {
-	autoreconf -vfi
```

Comment:
This is needed before runConfigure to set the correct paths ... re-add

Reply:
@Begasus - "alaviss - Please remove autoreconf from the recipe, as the patch is no longer necessary (merged to upstream since version 2.20)."

I re-added it for now.